aten/src/ATen/native/native_functions.yaml

```diff
@@ -7102,6 +7102,9 @@
 - func: can_cast(ScalarType from, ScalarType to) -> bool
   variants: function
 
+- func: mkldnn_delete_param(int param_handle, int param_type) -> bool
+  variants: function
```

Comment:
This is something that cannot be registered and has to be declared here?

Reply:
Due to that it is with no tensor params that will fail to dispatch when registered under mkldnn/CPU device, I've not changed to use torch.ops.mkldnn and CatchAll to get it work.